### ISSUE TITLE
CI support for dev-sec.io

### ISSUE THREAD

**Issue by artem-sidorenko** [dev-sec/dev-sec.github.io#19]
It would be nice to have auto publishing via CI:

- two repos:
   - `dev-sec.github.io`
   - `homepage`
- new github bot user
- bot user has dedicated r/w access to `dev-sec.github.io`
- `homepage` has Travis CI job, which automatically publishes the content to `dev-sec.github.io` for master builds

**Comment by artem-sidorenko** [comment]
@chris-rock I'll be happy to implement this if you assist me with right management on the github side

**Comment by chris-rock** [comment]
Yes, I have a deploy.sh script in the root already. We need to execute that once something is merged on src. I think we can tie this somehow with travis.

I would prefer if we keep the src and the build in the same repository. I already changed the default branch in this repository to src.

**Comment by artem-sidorenko** [comment]
> I would prefer if we keep the src and the build in the same repository. I already changed the default branch in this repository to src.

In this case you would give the push rights to the source of website to the CI, this is the point I'm not 100% happy with
